index.js

```diff
@@ -2,18 +2,14 @@ require('dotenv').config()
 
 const Discord = require('discord.js')
 const nemojis = require('node-emoji')
-const get = require('get-value')
 const client = new Discord.Client()
 
 client.on('message', msg => {
-  const members = get(msg, 'mentions.members')
-  if (!members) return
-
-  const jason = members.find(member => member.user.username === 'jasonetco')
-  if (!jason) return
+  if (!msg.mentions.users.has(client.user)) return
 
+  const jason = msg.guild.me
   const random = nemojis.random()
-  return jason.setNickname(`${random.emoji}man`, `${msg.author.username} mentioned Jason, so here we are.`)
+  return jason.setNickname(`${random.emoji}man`)
```

Comment:
Since the `jason` variable isn't really used anymore, want to just remove it and do:

```suggestion
  return msg.guild.me.setNickname(`${random.emoji}man`)
```